languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/style.xml

```diff
@@ -14362,7 +14347,7 @@ USA
                 <example><marker>leçon particulière</marker></example>
             </rule>
         </rulegroup>
-        <rule id="PORTER_FRUIT" name="porter fruit" tone_tags="clarity">
+        <rule id="PORTER_FRUIT" name="porter fruit" tone_tags="general">
```

Comment:
believe it or not, this rule has only TPs in diff.
improvement(unblocking): rules CENTRE-JARDIN,TOUR_A_FEU, TOUR_DEAU;POTEAU_DE_BARBIER and maybe ENSEMBLE_DE_PATIO; MEUBLE_DE_PATIO should have the same tag than those? 🥜

Reply:
Added :D